full-node/sov-prover-storage-manager/src/lib.rs

```diff
@@ -23,6 +26,9 @@ struct NewProverStorageManager<Da: DaSpec, S: MerkleProofSpec> {
 
     latest_snapshot_id: SnapshotId,
     block_hash_to_snapshot_id: HashMap<Da::SlotHash, SnapshotId>,
+
+    orphaned_snapshots: HashSet<SnapshotId>,
```

Comment:
Can you please add a comment on why do we need it?

Reply:
Done!